charts/stable/seafile/values.yaml

```
@@ -29,13 +29,49 @@ workload:
             DB_ROOT_PASSWD: "{{ .Values.mariadb.rootPassword }}"
             SEAFILE_SERVER_LETSENCRYPT: false
             SEAFILE_SERVER_HOSTNAME: ""
-            SEAFILE_ADMIN_EMAIL: me@example.com
-            SEAFILE_ADMIN_PASSWORD: a_very_secret_password
+            INIT_SEAFILE_ADMIN_EMAIL: me@example.com
+            INIT_SEAFILE_ADMIN_PASSWORD: a_very_secret_password
             FORCE_HTTPS_IN_CONF: false
+            JWT_PRIVATE_KEY: a_very_very_very_very_secret_key
+            CLUSTER_SERVER: "true"
+            CLUSTER_INIT_MODE: "true"
+            CLUSTER_INIT_MEMCACHED_HOST: |
+              {{ printf "%v-%v" .Release.Name "memcached" }}
```

```suggestion
            CLUSTER_INIT_MEMCACHED_HOST: {{ printf "%v-%v:%v" .Release.Name "memcached" "11211" }}
```
Lets also add the port.

---

you can't add the port, only the hostname can be changed: https://github.com/haiwen/seafile-docker/blob/master/scripts/scripts_12.0/bootstrap.py#L167

---

> you can't add the port, only the hostname can be changed: https://github.com/haiwen/seafile-docker/blob/master/scripts/scripts_12.0/bootstrap.py#L167

What the hell -_-. Ok then without the port.